package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
-    "gatsby": "^2.0.35",
+    "gatsby": "2.0.33",
```

Comment:
To be clear though, this happens in both versions, right?

Reply:
Yeap. Lost track slightly which exact version this error appears, but deffo 2.0.33 and later.